databroker/assets/sqlite.py

```diff
@@ -255,14 +255,18 @@ def find_one(self, query):
         # it up in an older one.
         for select in [SELECT_RESOURCE, OLD_SELECT_RESOURCE]:
             with cursor(self._conn) as c:
-                c.execute(select, (query['uid'],))
-                raw = c.fetchone()
+                raw = None
+                try:
+                    c.execute(select, (query['uid'],))
```

Comment:
Can you explain this change a bit more?

Reply:
It seems that if this runs on an old databroker it raises an exception rather than just returning `None` as expected. This now runs a try except so it can cycle through all the table schemas until it finds the right one or outright fails.

Reply:
Just to be sure we understand the theory of operation, what is the text of the exception that gets raised?

Reply:
`sqlite3.OperationalError: no such table: Resources_v2`